aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/options/AWSCognitoAuthConfirmSignInOptions.kt

```diff
@@ -53,6 +58,7 @@ data class AWSCognitoAuthConfirmSignInOptions internal constructor(
     class CognitoBuilder : Builder<CognitoBuilder?>() {
```

Comment:
is this in line with iOS ?

Reply:
it is inline with other option in Android repo.